tests/bal_functional_behaviour_suite.py

```diff
@@ -75,7 +75,7 @@ def test_when_preflight_called_then_results_delayed_by_specified_simulated_query
     def test_when_entityExists_called_then_results_delayed_by_specified_simulated_query_latency(
         self,
     ):
-        self.__check_simulated_latencies_without_callbacks(
+        self.__check_simulated_latencies_with_callbacks(
```

Comment:
[consider] Could just be `__check_simulated_latencies` now.

Reply:
Ooo good point.

Reply:
Fixed in f46cd624